📝 Document macOS user deletion over SSH (#80358)
ansible/ansible: docs/docsite/rst/reference_appendices/faq.rst

        $ export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES

.. _macos_as_a_target_faq:

Running on macOS as a target
----------------------------

> When executing Ansible against a system with macOS Monteray 12, macOS Ventura
> 13 or above, as a target machine, one might encounter the following error:

Reviewer: ```suggestion
When managing a system with macOS Monterey 12, macOS Ventura
13 or above over SSH, one might encounter the following error:
```

  .. error::
        "eDSPermissionError" DS Error: -14120 (eDSPermissionError)

> 

Reviewer: ```suggestion

This is a good indication that *Allow full disk access for remote users* has not been enabled.

```

        $ export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES

.. _macos_as_a_target_faq:

Running on macOS as a target
----------------------------

> When managing a system with macOS Monterey 12, macOS Ventura
> 13 or above over SSH, one might encounter the following error:

Reviewer 2: ```suggestion
When managing a system with macOS Monterey 12, macOS Ventura
13 or above over SSH, the following error can occur:
```

Reviewer 2: Just a style thing but I think the more formal tone can be a little distracting.

  .. error::
        "eDSPermissionError" DS Error: -14120 (eDSPermissionError)

This is a good indication that *Allow full disk access for remote users* has not been enabled.

.. seealso::

   For more details, check out `the official Apple user guide article

>    <https://support.apple.com/guide/mac-help/mchlp1066/mac#mchlp1b6a98a>`_.

Reviewer 2: It's not a huge deal but I consider it a best practice not to link to 3rd party pages like this from the docs to avoid link rot and maintenance.

Reviewer 2: I'll resolve this and get everything merged. I was just making a point and I'm sure this isn't the only 3rd party link in the docset. Thanks!

Author: OTOH, we'd end up having to maintain a copy of someone else's docs. I don't know which one is worse :man_shrugging:
